.travis.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
     - stage: tests
       os: linux
-      env: SWIFT_VERSION=4.2
+      env: SWIFT_VERSION=4.2.4
```

Comment:
Swift 4.2.0 on Linux has a difference (a bug, I assume) in the `JSONDecoder` implementation that causes the runner to fail parsing the spec test files. Later versions and swift on macOS don't seem to suffer from it, though. Bumping to 4.2.4 works on all variants, so I've done that here.